Tests/UnitTests/Android/app/src/main/cpp/CMakeLists.txt

```diff
@@ -8,14 +8,17 @@ project(UnitTestsJNI)
 get_filename_component(UNIT_TESTS_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../../.." ABSOLUTE)
 get_filename_component(REPO_ROOT_DIR "${UNIT_TESTS_DIR}/../.." ABSOLUTE)
 
+include("${UNIT_TESTS_DIR}/../../content.cmake")
```

Comment:
`content.cmake` includes iOS stuff which needs to happen before project. It's a bit weird to factor it out this way.

Reply:
include moved.